packages/vue/src/generators/library/library.ts

```diff
@@ -58,7 +58,7 @@ export async function libraryGenerator(tree: Tree, schema: Schema) {
   if (options.component) {
     tasks.push(
       await componentGenerator(tree, {
-        name: options.fileName,
```

Comment:
Not sure this needs to be removed: https://github.com/nrwl/nx/blame/master/packages/vue/src/generators/library/library.ts#L61

It's been there all along, it's not the one causing the issues, I think.

Reply:
But no harm in removing it either, it seems.

Reply:
It's just changing the component name from a prefixed version like `ProductProduct` to `Product`.